### PR TITLE
Add 'back to top' button to all  Resources and Support article-types

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR adds the "Scroll to top" button onto all of the article types in the Resources and Support website section. Currently it is only on the Multiple FAQ article type.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/16369

## Testing done
I included the component on each R&S page-type, did a `yarn build:content`, then navigated to an instance of each type of page and confirmed the scroll-top button is there and works.

I tested on -

1. `http://localhost:3001/resources/get-a-premium-my-healthevet-account/` (About)
2. `http://localhost:3001/resources/how-to-get-a-premium-ds-logon-account-online/` (Step by step)
3. `http://localhost:3001/resources/how-can-i-find-a-va-facility/` (Single QA)

We don't have any published articles for image lists or the checklist AFAIK so I couldn't test on those directly. But those will be tested in preview prior to publish anyway.

## Screenshots
image

## Acceptance criteria
- [ ] Scroll-to-top button exists

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/q_a.drupal.liquid [hunk 1]
```diff
@@ -70,6 +70,7 @@
       </div>
     </div>
   </main>
+  {% include "src/site/components/up_to_top_button.html" %}
```
Comment (ncksllvn) [inline src/site/layouts/q_a.drupal.liquid:73]: @zacharymorel I noticed that the FAQ Multiple layout includes several of these buttons where there is a heading.

```
<button type="button" class="vads-u-visibility--screen-reader">Back to top</button>
```

I'm out of the loop on the convo behind that. Do we need to add those buttons into each of these other layouts too?

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉

File: src/site/layouts/q_a.drupal.liquid (hunk 1 above)
Comment (zacharymorel) [reply src/site/layouts/q_a.drupal.liquid:73]: I'm also out of the loop, @youngfreezyVA would know more I think?

**Comment by ncksllvn** [comment]
I'll go ahead and merge this but @youngfreezyVA LMK if I'm missing something

**Pull request merged**